### PR TITLE
Fix/sk33lz/vagrant up

### PR DESCRIPTION
**Pull request by sk33lz**
This PR updates the probo-image-builder VM to generate a new docker image call `ubuntu-14.04-lamp-nightly.tar` and export it to the workspace NFS directory after a successful provision.

**To Test**

- [ ] vagrant up

- [ ] check to make sure you have a `ubuntu-14.04-lamp-nightly.tar` file in the `probo-image-builder/workspace` directory.

- [ ] Test inside the Probo VM against a known good repository.

There is still a bug that happens with the `wget -q -O - --header 'Authorization:token <*****>` that @lliss said he is making a different ticket for.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,12 @@
+.DS_Store
 .vagrant
-puppet/modules
+puppet/modules/
 *.swp
 .librarian
 .tmp
 builder/ubuntu-14.04_lamp.tar
 packer/packer_cache
+packer/scripts/.DS_Store
```
Comment (lliss) [inline .gitignore]: I'm pretty sure you can do a wildcard so you don't have to specify this one separately.

File: Vagrantfile [hunk 2]
```diff
@@ -16,7 +16,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # Sync all of the projects to your local machine
   # To enable, create a workspace directory in this directory
   if File.directory?("workspace")
-    config.vm.synced_folder "workspace", "/workspace" , type: "nfs"
+    config.vm.synced_folder "workspace", "/vagrant/workspace" , type: "nfs"
```
Comment (lliss) [inline Vagrantfile]: Why bother doing this? The entire repo is automatically mounted in the vm at `/vagrant`. So if this is causing trouble just remove it and grab the exported stuff out of the directory itself.

File: .gitignore (hunk 1 above)
Comment (sk33lz) [inline .gitignore]: Yeah, I thought about that after the commit...

**Comment by sk33lz** [comment]
I've added a wildcard for the .DS_Store file in .gitignore and reworked my code so that it just exports to the /vagrant directory. Testing again locally.

**Comment by sk33lz** [comment]
Saved docker image .tar file is now just in the root of the repo since /vagrant is mounted by default. This should be good to go.

**Pull request merged**